packages/core-js/internals/structured-clone.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+var isSymbol = require('../internals/is-symbol');
+var getOwnPropertyNames = require('../internals/object-get-own-property-names');
+var classof = require('../internals/classof');
+var getBuiltin = require('../internals/get-built-in');
+var isObject = require('../internals/is-object');
+var has = require('../internals/has');
+
+var Set = getBuiltin('Set');
+var Map = getBuiltin('Map');
+
+function createDataCloneError(message) {
+  if (typeof DOMException === 'function') {
+    return new DOMException(message, 'DataCloneError');
+  }
+  return new Error(message);
+}
+
+/**
+ * Tries best to replicate structuredClone behaviour.
+ *
+ * @param {WeakMap} weakmap cache map
+ * @param {any} value object to clone
+ */
```

Comment:
`core-js` not uses JSDoc, so let's follow the current style.

Reply:
I added it temporarily for VS Code. I'm going to remove it when I switch this PR to be ready for review.